modules/ROOT/examples/installation/post-installation-steps.sh

```diff
@@ -17,15 +31,24 @@ rootuser='root'
 # depending what you enter when running the script.
 # You have to run this script twice, one time to prepare installation and one time post installation
 
+# In case you upgrade an existing installation, your original directory will be renamed including a timestamp 
+
 # Example input
-# New install using mkdir:     n/n/n (create missing directories, setup permissions and ownership)
-# Upgrade using mkdir:         n/n/n (you move/replace data, apps-external and config.php manually, set setup permissions and ownership)
-# New install using links:     y/y/n (link existing directories, setup permissions and ownership)
-# Upgrade using links:         y/n/y (link existing directories, copy config.php, permissions and ownership are already ok)
-# Post installation/upgrade:   either n/n/n or y/y/n
-# Reset all perm & own:        either n/n/n or y/y/n
+# New install using mkdir:     n/n/n/n (create missing directories, setup permissions and ownership)
+# Upgrade using mkdir:         y/n/n/n (you move/replace data, apps-external and config.php manually, set setup permissions and ownership)
+# New install using links:     n/y/y/n (link existing directories, setup permissions and ownership)
+# Upgrade using links:         y/y/n/y (link existing directories, copy config.php, permissions and ownership are already ok)
+# Post installation/upgrade:   either n/n/n/n or n/y/y/n
+# Reset all perm & own:        either n/n/n/n or n/y/y/n
 
 echo
+read -p "Do you want to upgrade an existing installation (y/N)? " -r -e answer
+if echo "$answer" | grep -iq "^y"; then
```

Comment:
You could simplify the if/else to `(echo "$answer" | grep -iq "^y") && do_upgrade="y" || do_upgrade="n"` instead, as it's just a simple test + assignment.